src/Glpi/RichText/UserMention.php

```diff
@@ -261,4 +266,46 @@ public static function refreshUserMentionsHtmlToDisplay(string $content): string
 
         return $content;
     }
+
+    public static function isEnabled(): bool
+    {
+        $pro = Profile::getById($_SESSION['glpiactiveprofile']['id']);
+        return $pro->fields['use_mentions'] !== self::USER_MENTION_DISABLED;
+    }
+
+    public static function isMentionsRestricted(): bool
+    {
+        $pro = Profile::getById($_SESSION['glpiactiveprofile']['id']);
+        return $pro->fields['use_mentions'] === self::USER_MENTION_RESTRICTED;
+    }
```

Comment:
I do not see any usage of these two methods outside the current method. Splitting them forces the current profile to be loaded twice, meaning doing twice the same SQL query.

All this could probably be done directly in the `getRestrictedUsers` method, to save a SQL query.